content/docs/command-reference/exp/save.md

```diff
@@ -1,9 +1,15 @@
 # exp save
 
-Save the current project workspace as a [DVC experiment].
+Capture the current state of the <abbr>workspace</abbr> as a [DVC experiment].
 
 [dvc experiment]: /doc/user-guide/experiment-management/experiments-overview
 
+<admon type="info">
+
+See also `dvc exp run`, which includes this operation.
```

Comment:
Not sure if we want to make `exp run` so prominent here though. Should we move this note to the end of the description?